hack/make-rules/test-cmd-util.sh

```diff
@@ -2411,7 +2412,7 @@ run_multi_resources_tests() {
 runTests() {
   if [ -z "${SUPPORTED_RESOURCES}" ]; then
     echo "Need to set SUPPORTED_RESOURCES env var. It is a comma separated list of resources that are supported and hence should be tested. Set it to (*) to test all resources"
-    return
+    exit 1
```

Comment:
SUPPORTED_RESOURCES is not a comma separated list, it is a bash array.

We'll never hit this echo if it was unset, since we'd die on the set -e. I think it should be be `if [[ -z "${SUPPORTED_RESOURCES:-}" ]]` for the (wrong) comment to ever pop...